waspc/data/Generator/templates/server/src/session.js

```diff
@@ -0,0 +1,31 @@
+import cookieSession from 'cookie-session'
+import csrf from 'csurf'
+
+import config from './config.js'
+
+const sessionConfig = {
+  name: config.session.cookie.name,
+  secret: config.session.cookie.secret,
+  httpOnly: true,
+  signed: true,
+  maxAge: config.session.cookie.maxAge,
+}
+
+const csrfConfig = {
+  cookie: {
+    key: config.csrf.cookie.name,
```

Comment:
Here it is called key but we call it name, how is that?

Reply:
Their library just calls it key, but most users would assume it is the name.

http://expressjs.com/en/resources/middleware/csurf.html

> key - the name of the cookie to use to store the token secret (defaults to '_csrf').